app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class VotesController < ApplicationController
```

Comment:
Nope. we don't need a VotesController.

Voting is an action on a product. Votes are meaningless unless accessed through a user or a product.